platform-configs/include/protectli-common.robot

```diff
@@ -67,7 +67,7 @@ ${TYPE_VERIFICATION}=                           ${TRUE}
 ${EMMC_SUPPORT}=                                ${TRUE}
 ${CPU_TESTS_SUPPORT}=                           ${TRUE}
 ${L2_CACHE_SUPPORT}=                            ${TRUE}
-${L3_CACHE_SUPPORT}=                            ${TRUE}
+${L3_CACHE_SUPPORT}=                            ${FALSE}
```

Comment:
It worked on other platforms, do we want to disable it globally for Protectli, or only for VP2410?

Reply:
Probably only for VP2410.

Reply:
@pkubaj So I am looking forward for another version of this commit.

Reply:
When I tried adding L3_CACHE_SUPPORT=FALSE in VP2410, robotidy modified the file to:
```
diff --git a/platform-configs/protectli-vp2410.robot b/platform-configs/protectli-vp2410.robot
index a1e6d905eee6..14215a99b641 100644
--- a/platform-configs/protectli-vp2410.robot
+++ b/platform-configs/protectli-vp2410.robot
@@ -7,9 +7,6 @@ ${FLASH_SIZE}=                          ${8*1024*1024}
 ${WIFI_CARD}=                           ${EMPTY}
 ${WIFI_CARD_UBUNTU}=                    ${EMPTY}
 ${LTE_CARD}=                            ${EMPTY}
-${DEF_CORES}=                           4
-${DEF_THREADS}=                         1
-${DEF_CPU}=                             4
 ${DEF_ONLINE_CPU}=                      0-3
 ${DEF_SOCKETS}=                         1
 ${INITIAL_CPU_FREQUENCY}=               2000
@@ -37,6 +34,8 @@ ${DASHARO_CHIPSET_MENU_SUPPORT}=        ${TRUE}
 ${UEFI_PASSWORD_SUPPORT}=               ${TRUE}
 ${ME_STATICALLY_DISABLED}=              ${TRUE}
 
+${L3_CACHE_SUPPORT}=                    ${FALSE}
+
 
 *** Keywords ***
 Power On
@@ -54,7 +53,7 @@ Power On
     Power Cycle On
 
 Flash Protectli VP2410 Internal
-    Set Local Variable    ${IS_FLASH_CHIP_CONTENT_IDENTICAL}    ${FALSE}
+    Set Local Variable    ${is_flash_chip_content_identical}    ${FALSE}
     Power On
     Boot Operating System    ubuntu
     Login To Linux
```

Because:
```
Autofixing: Removing lines containing unused variables from platform-configs/protectli-vp2410.robot
Found 3 defined variables not used in platform-configs/protectli-vp2410.robot:
${DEF_CORES}
${DEF_CPU}
${DEF_THREADS}
```

Those `DEF_*` parameters are definitely necessary.

Reply:
@macpijan we've just renamed these variables: https://github.com/Dasharo/open-source-firmware-validation/commit/c578a45d688aba0aab3584b5c3aa0fe6b69700de

Reply:
Renamed.